internal/rollup/rollup.config.js

```diff
@@ -0,0 +1,35 @@
+const rollup = require('rollup');
```

Comment:
put a header that this file is generated from somewhere?

Reply:
👍